src/reducer.js

```diff
@@ -0,0 +1,14 @@
+import { LP_API_ACTION } from './actions'
+import { get } from './utils'
+
+export default function (state={}, { type, payload }) {
+  if (type !== LP_API_ACTION) return state
+  const { key, status } = payload
+  return { ...state, [key]: status }
+}
+
+export function selectStatus (key, state, slice='api') {
```

Comment:
I'm gonna leave this for now- I think the overwhelming use case will be to not use a custom slice anyway.